src/components/Navigation/TopBar/Help.tsx

```diff
@@ -41,13 +41,13 @@ const Spacer = styled.div`
   display: flex;
 `;
 
-export interface HelpProps {
+type HelpType = {
```

Comment:
Why does it need to be a type? 
I think we "normally" use interface for the props of a component, but they are basicly the same i believe, so was just curious if there was a specific reason 😄

Reply:
I don't know really. I copied this way of doing to from some other component that uses forwardRef. I got it to work locally, but forgot to change to type when I put it into amplify-components. Now when I import Help in PWEX it gets error Type '{ children: Element[]; applicationName: string; ref: MutableRefObject<HTMLDivElement | null>; }' is not assignable to type 'IntrinsicAttributes & HelpProps'.
  Property 'children' does not exist on type 'IntrinsicAttributes & HelpProps'.